src/vs/workbench/services/issue/electron-sandbox/issueService.ts

```diff
@@ -56,6 +59,22 @@ export class NativeIssueService implements IWorkbenchIssueService {
 			const result = await this.getIssueTemplate(request.extensionId, CancellationToken.None);
 			ipcRenderer.send(request.replyChannel, result);
 		});
+		ipcRenderer.on('vscode:triggerReporterStatus', async (event, arg) => {
+			const extensionId = arg.extensionId;
+			const extension = await this.extensionService.getExtension(extensionId);
+			if (extension) {
+				const activationEvents = this._activationEventReader.readActivationEvents(extension);
+				for (const activationEvent of activationEvents) {
+					if (activationEvent === 'onIssueReporterOpened') {
+						const eventName = `onIssueReporterOpened:${ExtensionIdentifier.toKey(extension.identifier)}`;
+						await this.extensionService.activateById(extension.identifier, { startup: false, extensionId: extension.identifier, activationEvent: eventName });
```

Comment:
Can this throw if the extension it is activating throws an error in its activation? If so, you should probably catch that and send the default value rather than having timeout handle it